platform_generator.R

```diff
@@ -132,6 +228,11 @@ if (TRUE %in% is.nan(site_to_SE_bandwidth$AvgBandwidth)) {
     max(transfers[transfers$UpDown != 2,]$Bandwidth)
 }
 
+if (TRUE %in% is.na(site_to_SE_bandwidth$AvgBandwidth)) {
+  site_to_SE_bandwidth[is.na(site_to_SE_bandwidth$AvgBandwidth),]$AvgBandwidth <- 
+    max(transfers[transfers$UpDown != 2,]$Bandwidth)
+}
+
 if (TRUE %in% is.infinite(site_to_SE_bandwidth$MaxBandwidth)) {
   site_to_SE_bandwidth[is.infinite(site_to_SE_bandwidth$MaxBandwidth),]$MaxBandwidth <- 
```

Comment:
This block (is.nan, is.na, is.infinite) is duplicated before and after the max correction. One has to be removed